packages/block-library/src/pattern/edit.js

```diff
@@ -52,7 +57,29 @@ const PatternEdit = ( { attributes, clientId, setAttributes } ) => {
 		innerBlocks,
 	] );
 
-	const blockProps = useBlockProps();
+	useEffect( () => {
+		const alignments = [ 'wide', 'full' ];
+		const blocks =
+			syncStatus === 'synced' ? selectedPattern?.blocks : innerBlocks;
+		// Determine the widest setting of all the contained blocks.
+		const widestAlignment = blocks.reduce( ( accumulator, block ) => {
```

Comment:
When a sync'd block first loads, this throws an error as I don't think the selected pattern has been returned yet. If we can tighten it up, it would also protect against a pattern that no longer exists.

<img width="996" alt="Screenshot 2023-04-27 at 6 10 28 pm" src="https://user-images.githubusercontent.com/60436221/234800624-4de01a48-7998-405d-a6cf-8891808d8a1e.png">